src/utils/test/util.spec.ts

```diff
@@ -155,18 +155,30 @@ describe('util', () => {
       // improperly formatted JSON - note the lack of ':'
       const diagnostic = util.parsePackageJson('{ "someJson" "value"}', mockPackageJsonPath);
 
+      // due to missing colon in input
+      const messageText =
```

Comment:
FWIW, I think that the build will fail due to this variable and `expectedDiagnostic` no longer being used.